x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -172,3 +172,35 @@ export const cleanupCredentials = (packagePolicy: NewPackagePolicy | UpdatePacka
   // nothing to do, return unmutated policy
   return packagePolicy;
 };
+
+export const getBenchmarkCisName = (benchmarkId: string) => {
+  switch (benchmarkId) {
+    case 'cis_k8s':
+      return 'CIS Kubernetes';
+    case 'cis_azure':
+      return 'CIS Azure';
+    case 'cis_aws':
+      return 'CIS AWS';
+    case 'cis_eks':
+      return 'CIS EKS';
+    case 'cis_gcp':
+      return 'CIS GCP';
+  }
+  return 'undefined';
```

Comment:
returning a string `undefined` is a bit confusing here. What is the reason for not returning `null` or throwing an error if we don't want to allow this to happen?